utility/environment_check.py

```diff
@@ -0,0 +1,133 @@
+"""
+Util for environment check befora and after test to compare and find stale
+leftovers
+"""
+import logging
+import pytest
+
+from ocs import ocp, constants, exceptions
+from deepdiff import DeepDiff
+
+log = logging.getLogger(__name__)
```

Comment:
@RazTamir I see that log logger is not used anywhere, didn't want you print some info to the log?